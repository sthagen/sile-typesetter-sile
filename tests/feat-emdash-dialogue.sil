\begin[papersize=a7]{document}
\nofolios
\neverindent
\font[size=22pt]
\set[parameter=linebreak.emergencyStretch, value=50pt]

% Affect paragraph-leading em-dash but not later...
— Lorem ipsum dolor sit. — Sed non risus.

— Non.

\set[parameter=typesetter.fixedSpacingAfterInitialEmdash, value=false]

— Lorem ipsum dolor sit. — Sed non risus.

\end{document}
