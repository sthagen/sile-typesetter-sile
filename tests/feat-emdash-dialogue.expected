Set paper size 	209.7637818	297.6377985
Begin page
Mx 	10.4882
My 	31.5323
Set font 	Gentium Plus;22;400;;normal;;;LTR
T	179 w=16.8545	(—)
Mx 	33.1564
T	47 82 85 72 80 w=58.0830	(Lorem)
Mx 	102.5788
T	76 83 86 88 80 w=55.4404	(ipsum)
Mx 	169.3586
T	71 82 w=22.5049	(do)
Mx 	191.8635
T	16 w=7.4121	(-)
Mx 	10.4882
My 	57.9323
T	79 82 85 w=25.7383	(lor)
Mx 	49.1531
T	86 76 87 w=22.0322	(sit)
Mx 	71.1853
T	17 w=5.0381	(.)
Mx 	89.1500
T	179 w=16.8545	(—)
Mx 	118.9310
T	54 72 71 w=32.0762	(Sed)
Mx 	163.9338
T	81 82 81 w=35.3418	(non)
Mx 	10.4882
My 	84.3323
T	85 76 86 88 86 w=43.3232	(risus)
Mx 	53.8114
T	17 w=5.0381	(.)
Mx 	10.4882
My 	110.7323
T	179 w=16.8545	(—)
Mx 	33.1564
T	49 82 81 w=37.4795	(Non)
Mx 	70.6358
T	17 w=5.0381	(.)
Mx 	10.4882
My 	137.1323
T	179 w=16.8545	(—)
Mx 	36.8402
T	47 82 85 72 80 w=58.0830	(Lorem)
Mx 	104.4207
T	76 83 86 88 80 w=55.4404	(ipsum)
Mx 	169.3586
T	71 82 w=22.5049	(do)
Mx 	191.8635
T	16 w=7.4121	(-)
Mx 	10.4882
My 	163.5323
T	79 82 85 w=25.7383	(lor)
Mx 	49.1531
T	86 76 87 w=22.0322	(sit)
Mx 	71.1853
T	17 w=5.0381	(.)
Mx 	89.1500
T	179 w=16.8545	(—)
Mx 	118.9310
T	54 72 71 w=32.0762	(Sed)
Mx 	163.9338
T	81 82 81 w=35.3418	(non)
Mx 	10.4882
My 	189.9323
T	85 76 86 88 86 w=43.3232	(risus)
Mx 	53.8114
T	17 w=5.0381	(.)
End page
Finish
